src/Security/Security/help/Get-AzSqlVulnerabilityAssessmentBaselineRule.md

```diff
@@ -0,0 +1,209 @@
+```
+Get-AzSqlVulnerabilityAssessmentBaselineRule -RuleId <String> -ResourceId <String> -WorkspaceId <String>
+ -Server <String> -Database <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### OnPremMachines
+```
+Get-AzSqlVulnerabilityAssessmentBaselineRule -RuleId <String> -WorkspaceId <String> -Server <String>
+ -Database <String> -ComputerName <String> -VMUUID <String> -AgentId <String> -WorkspaceResourceId <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get SQL Vulnerability Assessment baseline (rule level).
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzSqlVulnerabilityAssessmentBaselineRule -ResourceId /subscriptions/f26d1f13-67d5-4ad6-9012-67ca12d2436f/resourcegroups/ahmadtesting/providers/microsoft.operationalinsights/workspaces/ahabas-workspace/onPremiseMachines/ahabas-dev01.middleeast.corp.microsoft.com_49640166-652f-4ee6-b48b-cfb840b8afe2_4c4c4544-0030-4b10-8039-b8c04f4a3332 -WorkspaceId  ba7c9d0e-a6e3-4997-b575-cf7a18a98a49 -Server AHABASDEV01SRV -Database master -RuleId "VA2108"
+ 
+ Results                 Id
+-------                 --
+{dbo db_owner SQL_USER} /subscriptions/f26d1f13-67d5-4ad6-9012-67ca12d2436f/resourcegroups/ahmadtesting/providers/microsoft.operationalinsights/workspaces/ahabas-workspace/onpremisemachines/ahabas-dev01.middleeast.corp.microsoft.com_49640166-652f-4ee6-b48b-cfb840b8afe2_4c4c4544-0030-4b10-8039-b8c04f4a3332/sqlServe…
+
+```
+
+
+## PARAMETERS
+
+### -AgentId
+Agent id - on premise parameter
+
+```yaml
+Type: System.String
+Parameter Sets: OnPremMachines
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ComputerName
+Computer full name - on premise parameter
+
+```yaml
+Type: System.String
+Parameter Sets: OnPremMachines
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Database
+Database name
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceId
+ID of the security resource that you want to invoke the command on.
+
+```yaml
+Type: System.String
+Parameter Sets: ResourceId
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RuleId
+Vulnerability Assessment rule id
```

Comment:
```suggestion
Vulnerability Assessment rule ID
```
Please change all id to ID to unify format.

Reply:
pending